backend/src/main/java/com/soen341/instagram/controller/ProfileController.java

```diff
@@ -1,92 +1,82 @@
 package com.soen341.instagram.controller;
 
+import org.modelmapper.ModelMapper;
 // Spring Boot
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PutMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RestController;
 
 // Project
 import com.soen341.instagram.dto.account.ProfileRequestDTO;
+import com.soen341.instagram.model.Account;
 import com.soen341.instagram.service.impl.ProfileService;
 
 @RestController
-public class ProfileController {
-	
+public class ProfileController
+{
+
 	@Autowired
 	private ProfileService profileService;
-	
+	@Autowired
+	private ModelMapper modelMapper;
+
 // For now we are not allowing the user to change their username
 //	@PutMapping(value = "account/profile/usernameUpdate")
 //	public void setUsername(@RequestBody final UpdateProfileRequestDTO profileRequest)
 //	{
 //		String username = profileRequest.getUsername();
 //		profileService.setUsername(username);
 //	}
-	
+
 	@PutMapping(value = "account/profile/biographyUpdate")
 	public void setBiography(@RequestBody final ProfileRequestDTO profileRequest)
 	{
 		String biography = profileRequest.getBiography();
 		profileService.setBiography(biography);
 	}
-	
+
 	@PutMapping(value = "account/profile/emailUpdate")
 	public void setEmail(@RequestBody final ProfileRequestDTO profileRequest)
 	{
 		String email = profileRequest.getEmail();
 		profileService.setEmail(email);
 	}
-	
+
 	@PutMapping(value = "account/profile/displayNameUpdate")
 	public void setDisplayName(@RequestBody final ProfileRequestDTO profileRequest)
 	{
 		String displayName = profileRequest.getDisplayName();
 		profileService.setDisplayName(displayName);
 	}
-	
+
 	@PutMapping(value = "account/profile/nameUpdate")
 	public void setName(@RequestBody final ProfileRequestDTO profileRequest)
 	{
 		String firstName = profileRequest.getFirstName();
 		String lastName = profileRequest.getLastName();
-		
+
 		profileService.setName(firstName, lastName);
 	}
-	
+
 //TODO: Implement changing profile pictures
```

Comment:
We should have a discussion on what we are planning to do with the profile picture. Currently our user page does not even support one.

Reply:
I'm not sure if it was officialized or not, but I know we let go of that idea because of its complexity (need to implement a new table in the backend + create a new controller and service, as well as to implement it in the frontend aftwards). It could be something we add after the completion of the project, but I don't know if we'd be able to implement it within our current timeframe.